### PR TITLE
[CONTRUBITING] Add section on using a secondary devel directory

### PR DESCRIPTION
**Pull request by samcv**
Add a section on how to use a separate directory than your normal
ZDOTDIR for development with its own zprezto.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -35,6 +35,24 @@ improve its performance, do not hesitate to fork and send pull requests.
    - Avoid `eval` if possible.
    - Prefer `zstyle` over environment variables for configuration.
 
+#### Using an Alternative zprezto Directory
+
+To work on zprezto without messing with your current configuration:
+
+    mkdir devel-zprezto
```
Comment (belak) [inline CONTRIBUTING.md]: Please use fenced code blocks here (the 3 back ticks). We had issues with these blocks rendering improperly when uses spaces to trigger a code block.

**Comment by samcv** [comment]
Fixed the non fenced codeblock.

It was getting annoying working with multiple git branches. I tried several zsh command line options first first. It turned out to be easier than I had guessed when I tried to just set ZDOTDIR :)

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -35,6 +35,26 @@ improve its performance, do not hesitate to fork and send pull requests.
...
+setopt EXTENDED_GLOB
+for rcfile in "${ZDOTDIR:-$HOME}"/.zprezto/runcoms/^README.md(.N); do
+    ln -s "$rcfile" "${ZDOTDIR:-$HOME}/.${rcfile:t}"
+done
+```
+
+Then to start zsh in this development environment you will run:
+
+    ZDOTDIR=/path/to/devel-zprezto zsh
+
```
Comment (indrajitr) [inline CONTRIBUTING.md]: Maybe fence this one too?

Comment (samcv) [inline CONTRIBUTING.md]: Good catch. Fixed :)

**Comment by belak** [comment]
Thanks!

**Pull request merged**